src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -345,8 +345,14 @@ cldnn::format find_data_format(dnnl::memory::desc desc) {
                     break;
                 }
             }
-            if (is_match)
-                return static_cast<format::type>(fmt_idx);
+
+            if (is_match) {
+                // b_fs_yx_fsv2 has dnnl::memory::format_tag::undef , so select byfx instead of b_fs_yx_fsv2.
+                if (static_cast<format::type>(fmt_idx) == format::type::b_fs_yx_fsv2)
+                    return format::type::byxf;
+                else
+                    return static_cast<format::type>(fmt_idx);
+            }
         }
     }
 
```

Comment:
random spot) could you add some functional test for fsv2 case?

Reply:
Added unit test. Thanks!